extensions/sql-database-projects/src/dialogs/utils.ts

```diff
@@ -30,6 +31,14 @@ export function getAgreementDisplayText(agreementInfo: AgreementInfo): string {
 	return constants.eulaAgreementText(agreementInfo.link!.text);
 }
 
+export function getPublishServerName(target: string): string {
+	let name: string = constants.SqlServerName;
+	if (target === constants.targetPlatformToVersion.get(SqlTargetPlatform.sqlAzure)) {
+		name = constants.AzureSqlServerName;
+	}
+	return name;
```

Comment:
can simplify to 
`return target === constants.targetPlatformToVersion.get(SqlTargetPlatform.sqlAzure) ? constants.AzureSqlServerName : constants.SqlServerName;`